lib/bundles/inspec-compliance/cli.rb

```diff
@@ -79,10 +106,8 @@ def profiles
     def exec(*tests)
       config = Compliance::Configuration.new
       return if !loggedin(config)
-
       # iterate over tests and add compliance scheme
-      tests = tests.map { |t| 'compliance://' + t }
-
+      config['automate'][0] ? tests = tests.map { |t| 'automate://' + t } : tests = tests.map { |t| 'compliance://' + t }
```

Comment:
I would prefer if we could stick to `compliance://` no matter if the profiles are stored in Chef Compliance or Chef Automate